_config.yml

```diff
@@ -40,3 +41,8 @@ defaults:
 tipue_search:
   include:
     pages: true
+collections:
+  custom_org:
+    output: true
+    collection_dir: /aip/custom_org
+    permalink: /:collection/:name
```

Comment:
Adding the custom org as a collection, but still under the aip directory.

Not sure if I will keep it, or if it will be necessary. What I like about it is that it enforces the permalink to be connected to the collection name. Assuming the scope == collection name, then I can always count on the permalink following this format and know if an extension of an AIP exists.

Reply:
Did you work around the problem of having to manually specify `layout` on collections?
(Note: I did find a workaround for the linking issues.)

Reply:
TLDR: Relative links like `./0001.md` stop working in collections for some reason. There is a Liquid directive that will do the right thing (you just lose GitHub linkability, which I do not mind as much as I used to...).